src/Pkg.jl

```diff
@@ -671,6 +672,7 @@ const precompile_script = """
     Pkg.add("TestPkg")
     Pkg.develop(Pkg.PackageSpec(path="TestPkg.jl"))
     Pkg.add(Pkg.PackageSpec(path="TestPkg.jl/"))
+    Pkg.try_prompt_pkg_add(Symbol[:notapackage])
```

Comment:
Won't this block on `prompt`?

Reply:
No because `:notapackage` won't be found. But you did make me realize it's missing a module reference
```
julia> Pkg.Operations.try_prompt_pkg_add(Symbol[:notapackage])
false
```

Reply:
Ok. Is the lag noticable without this?

Reply:
I think so